docs/guide/start/managing-cli.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Monitoring and Managing Applications
+title_in_menu: Monitoring and Managing Applications
+layout: website-normal
+menu_parent: index-cli.md
+children:
+- { section: Applications } 
+- { section: Entities } 
+- { section: Sensors  } 
+- { section: Effectors  } 
+- { section: Activities } 
+---
+
+
+
+So far we have touched on Brooklyn's ability to *deploy* an application blueprint to a cloud provider, but this just 
+the beginning. The sections below outline how to manage the application that has been deployed.
+
+## Applications
+
+Having created the application we can query its status.  We can find a summary of all deployed apps:
+{% highlight bash %}
+$ br app
```

Comment:
Nitpick: Here and following, inconsistent use of the short version of commands etc. (EG "app" for "application"). I would say that the first use in any document should be the full version, and subsequent uses may or should be shortened.